guide/src/class.md

```diff
@@ -460,13 +478,54 @@ use pyo3::types::{PyDict, PyTuple};
 #
 #[pymethods]
 impl MyClass {
-    #[args(arg1=true, args="*", arg2=10, args3="\"Hello\"", kwargs="**")]
-    fn method(&self, arg1: bool, args: &PyTuple, arg2: i32, arg3: &str, kwargs: Option<&PyDict>) -> PyResult<i32> {
-        Ok(1)
+    #[new]
+    #[args(num="-1", debug="true")]
+    fn new(obj: &PyRawObject, num: i32, debug: bool) {
+        obj.init({
+            MyClass {
+                num,
+                debug,
+            }
+        });
+    }
+    #[args(num="10", debug="true", py_args="*", name="\"Hello\"", py_kwargs="**")]
+    fn method(&mut self, num: i32, debug: bool, name: &str, py_args: &PyTuple,
+                py_kwargs: Option<&PyDict>) -> PyResult<String> {
+        self.debug = debug;
+        self.num = num;
+        Ok(format!("py_args={:?}, py_kwargs={:?}, name={}, num={}, debug={}",
+                    py_args, py_kwargs, name, self.num, self.debug))
+    }
+    fn make_change(&mut self, num: i32, debug: bool) -> PyResult<String> {
+        self.num = num;
+        self.debug = debug;
+        Ok(format!("num={}, debug={}", self.num, self.debug))
     }
 }
-```
 
+#[pymodule]
+fn mymodule(_py: Python, m: &PyModule) -> PyResult<()> {
+    m.add_class::<MyClass>()?;
+    Ok(())
+}
+```
+N.B. the position of the `"*"` argument (if included) controls the system of handling positional and keyword arguments. In Python:
```

Comment:
This Python code example is really good :+1:

Reply:
Hi, all good suggestions. Hopefully I've incorporated them all.